modules/table/src/main/java/org/apache/ignite/internal/table/distributed/disaster/GroupUpdateRequest.java

```diff
@@ -304,14 +304,19 @@ private static CompletableFuture<Integer> partitionUpdate(
 
         Assignment nextAssignment = nextAssignment(localPartitionStateMessageByNode, partAssignments);
 
+        boolean isNextEqualsToRestoredAsPlannedAssignments = partAssignments.size() == 1 && partAssignments.contains(nextAssignment);
```

Comment:
I believe that something like `isProposedPendingEqualsProposedPlanned` would be clearer. Up to you.

Reply:
Agree